Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1149,6 +1149,10 @@ def identityFS(wf):
                           '--era'         : 'Run2_2016',
                         }
 
+# LHE-GEN-SIM with DQM
+step1LHEGenSimDQM = merge([{'-s':'LHE,GEN,SIM','--datatier'    : 'GEN-SIM,LHE,DQMIO','--eventcontent': 'LHE,RAWSIM,DQM'},step1LHEGenSimDefault])
```

Comment:
Hi @Andrej-CMS , this is a release validator from GEN side. While doing the recent release validation campaign, I found that the workflow 535 generates corrupted DQM files, causing comparison plots unavailable. As far as I know, for this GEN-only relval workflow, SIM step is not needed and this should be replaced by VALIDATION:genvalid step like other GEN-only workflows to make relval works. Could you let me fix this one? I hope that this doesn't violate your needs.

Reply:
Hi @SanghyunKo ,

You can change the workflow if you want. no problem.

Reply:
Thanks @Andrej-CMS , I'll make it fixed then.